src/tests/ftest/control/daos_admin_privileged.yaml

```diff
@@ -9,6 +9,6 @@ server_config:
   port: 10001
   servers:
     bdev_class: nvme
-    bdev_list: ["0000:81:00.0","0000:da:00.0"]
+    bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
```

Comment:
Nice test.

I wonder if we should invalidate all of the `bdev_list` values in all of the `.yaml` files like this to help guide people that these need to be filled in one way or another, specific to their systems.  Otherwise valid looking values (such as they are currently) make it less obvious, IMHO.

Reply:
Yes I do agree.  I'll create a new story for this change once this lands.